Makefile

```
@@ -34,8 +34,9 @@ all: configure build sign-dmg verify-signed
 apple: notarize staple verify-notarization
 
 configure:
```

checkmake (0.2.2)

[warning] Target "configure" should be declared PHONY.

(phonydeclared)